.mailmap

```diff
@@ -0,0 +1,11 @@
+Ares Meroueh <ares.m@asidatascience.com> ameroueh <ares.meroueh@gmail.com>
+Elias Benussi <elias@faculty.ai> Elias Benussi <4412300+eliasbenussi@users.noreply.github.com>
+Elias Benussi <elias@faculty.ai> Elias Benussi <elias@asidatascience.com>
+Hailey Fong <hailey.fong@faculty.ai> Hailey Fong <haileyfong1997@gmail.com>
+Hailey Fong <hailey.fong@faculty.ai> Hailey Fong <haileyfong@asiadmins-MacBook-Pro-2.local>
+Jan Freyberg <jan@asidatascience.com> Jan Freyberg <jan.freyberg@gmail.com>
```

Comment:
Should we map Jan's contributions to his person email since he's no longer at the company?